src/renderer/widgets/WalletDetails/ui/components/WalletConnectAccounts.tsx

```diff
@@ -25,11 +25,11 @@ type Props = {
 export const WalletConnectAccounts = ({ wallet, accounts }: Props) => {
   const { t } = useI18n();
 
+  const chains = useUnit(networkModel.$chains);
   const reconnectStep = useUnit(wcDetailsModel.$reconnectStep);
 
-  // TODO: Rework with https://app.clickup.com/t/8692ykm3y
   const accountsList = useMemo(() => {
-    const sortedChains = chainsService.getChainsData({ sort: true });
+    const sortedChains = chainsService.sortChains(Object.values(chains));
```

Comment:
Chains are already sorted in `networkModel`
line 12 `const chains = chainsService.getChainsMap({ sort: true });`